src/main/kotlin/ai/digital/integration/server/common/util/KubeCtlHelper.kt

```diff
@@ -71,8 +71,8 @@ open class KubeCtlHelper(val project: Project, isOpenShift: Boolean = false) {
             user,
             getClusterServer(cluster),
             getClusterCertificateAuthorityData(cluster),
-            getUserClientCertificateData(user),
-            getUserClientKeyData(user)
+            if(!skip) getUserClientCertificateData(user) else null,
```

Comment:
Please always format the code.
I see even without Intellij IDEA that it is not formatted, there is always a space between if and bracket. It is not the single place where formatting is not correct.

Reply:
done